### DISCUSSION TITLE
Finding MX Id in depthai C++ API

### DISCUSSION THREAD

**Discussion by vsranjitroshan** [luxonis/depthai#764] [General]
Whenever I run device.findAllAvaiableDevices() and get their information, c++ always returns the IP address instead of the MX ID of the OAK-D PoE camera connected. I need those because only they are unique. Python API gives the MX ID correctly. How can I obtain the MX ID information of the connected OAK camera's in c++?

**Comment by Luxonis-Brandon** [comment]
Sorry about the trouble.  Asking.

**Comment by SzabolcsGergely** [comment]
@vsranjitroshan 
`device.getDeviceInfo().getMxId()`
